ssl/record/methods/dtls_meth.c

```diff
@@ -251,7 +254,9 @@ static int dtls_process_record(OSSL_RECORD_LAYER *rl, DTLS_BITMAP *bitmap)
      * Length setting.
      */
     if (rr->length > rl->max_frag_len) {
-        RLAYERfatal(rl, SSL_AD_RECORD_OVERFLOW, SSL_R_DATA_LENGTH_TOO_LONG);
+        /* As per RFC 9147 4.5.2: Silently drop the packet in case of invalid length. */
+        rr->length = 0;
+        rl->packet_length = 0;
```

Comment:
rfc6066, section 4:
> When this extension is used
> with Datagram Transport Layer Security (DTLS), implementations SHOULD
> NOT generate record_overflow alerts unless the packet passes message
> authentication.
   
So actually this one should generate an alert, don't you agree?
If so please add a reference for rfc6066

Reply:
Good catch, I understand the same thing, will revert this change. Thank you